lab-james/models/client.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const uuid = require('node-uuid');
```

Comment:
**TODO FOR LOST POINTS**

You need to ``npm i -S node-uuid`` or else nothing works. I won't review thoroughly until that's done.